page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/Tag.java

```diff
@@ -0,0 +1,14 @@
+package ru.sbtqa.tag.pagefactory;
+
+import org.junit.AfterClass;
+import ru.sbtqa.tag.pagefactory.environment.Environment;
+
+public class Tag {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=103&issues=AWbTLdVJLpqqSUlcaTAo&open=AWbTLdVJLpqqSUlcaTAo)

[ISSUE_KEY]: AWbTLdVJLpqqSUlcaTAo